scaffolding-chef-infra/plan.ps1

```diff
@@ -5,10 +5,10 @@ $pkg_version="0.6.0"
 $pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 $pkg_license=("Apache-2.0")
 $pkg_upstream_url="https://www.chef.sh"
-$pkg_build_deps=@(
+$pkg_deps=@(
```

Comment:
We need both `git` and `chef-dk` at run time?

Reply:
Not really, the *scaffolding* needs git and chef-dk at runtime (which is during the upstream's build). In the Linux scaffolding we get around this by using Scaffolding-Load(), but Scaffolding-Load() has a bug with Windows, causing it to not function properly. So this is the workaround for now.

Reply:
Sounds good to me. I won't block a merge over it, but I'd like to add a TODO with a link to an issue just so it is tracked.